components/ResourceCreation/PatientCreation.tsx

```diff
@@ -34,6 +45,46 @@ function PatientCreation({
   const measurementPeriod = useRecoilValue(measurementPeriodState);
   const [selectedPatient, setSelectedPatient] = useRecoilState(selectedPatientState);
   const [isConfirmationModalOpen, setIsConfirmationModalOpen] = useState(false);
+  const measureBundle = useRecoilValue(measureBundleState);
+  const [showCalculation, setShowCalculation] = useState(false);
+  const { classes } = useStyles();
+
+  // Function to wrap the calculate function and catch errors in fqm-execution
+  const clickCalculateButton = async (id: string | null) => {
+    try {
+      await calculate(id);
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
+  // Function to calculate the selected patient's measure report
+  const calculate = async (id: string | null) => {
+    const options: CalculatorTypes.CalculationOptions = {
+      calculateHTML: true,
+      calculateSDEs: false,
```

Comment:
I remember we set this to false because of a bug in fqm-execution. This was actually fixed (not released yet but hopefully will be soon), can we just add a `TODO` comment above this to flip `calculateSDEs` to `true` once a new `fqm-execution` version is released/the dependency is updated? I don't wanna forget